docs/index.rst

```diff
@@ -79,111 +78,44 @@ Getting Started
   <http://www.plasmapy.org/>`_
 * `Using astropy.units <http://docs.astropy.org/en/stable/units/>`_
 
-.. _toplevel-user-documentation:
-
-******************
-User Documentation
-******************
-
-.. _toplevel-plasma-parameters:
-
-Formulary
----------
-
-The `~plasmapy.formulary` subpackage aims to cover the `NRL Plasma Physics
-Formulary <https://www.nrl.navy.mil/ppd/content/nrl-plasma-formulary>`_.
-
-.. toctree::
-   :maxdepth: 1
-
-   formulary/index
-
-.. _toplevel-experimental-tools:
-
-Experimental Tools
-------------------
-
-The `~plasmapy.diagnostics` package is in the early stages of
-development.
-
 .. toctree::
    :maxdepth: 1
+   :caption: Package features
+   :hidden:
 
    diagnostics/index
-
-.. _toplevel-data-structures:
-
-Data Structures and Simulation
-------------------------------
-
-.. toctree::
-    :maxdepth: 1
-
-    plasma/index
-    simulation/particletracker
-
-.. _toplevel-particle-data:
-
-Particle Data
--------------
-
-.. toctree::
-    :maxdepth: 1
-
-    particles/index
-
-.. _toplevel-utilities:
-
-Utilities
----------
-
-.. toctree::
-   :maxdepth: 1
-
+   formulary/index
+   particles/index
+   simulation/particletracker
+   plasma/index
    utils/index
 
-.. _toplevel-examples:
-
-Examples
---------
 .. toctree::
-    :maxdepth: 1
-
-    auto_examples/index
-
-.. _toplevel-development-guide:
+   :maxdepth: 3
+   :caption: About
+   :hidden:
 
-*****************
-Development Guide
-*****************
-
-The :ref:`plasmapy-development-guide` contains information on how to
-contribute to PlasmaPy, along with guidelines for code, testing, and
-documentation.
+   development/index
+   development/code_guide
+   development/doc_guide
+   development/testing_guide
+   development/release_guide
 
 .. toctree::
-    :maxdepth: 1
-    :hidden:
-
-    development/index
+   :maxdepth: 3
+   :caption: About
+   :hidden:
 
-.. _toplevel-project-details:
+   about/index
```

Comment:
```suggestion
   about/credits
```

Linking to the `about/index` is kind of redundant since the rest of the links already show up in other places in the side bar.